### DISCUSSION TITLE
🙋 Grafieken mainpaige blijven leeg

### DISCUSSION THREAD

**Discussion by Marnixvdveen** [dsmrreader/dsmr-reader#1525] [Questions & answers / How-to's] [answered]
### Description

Sinds iets meer dan een week blijven mijn grafieken op de mainpage leeg. Ik heb mijn NUC al gereset, updates uitgevoerd, DSMR herstart met SupervisorCTL restart all, datalogger sleep van 0,5 naar 2 gezet en ik bewaar nog maar 1 maand aan uitgebreide informatie.

Ik krijg wel live informatie binnen, dus ik zie wel actueel verbruik maar de grafieken blijven leeg. Ik heb een vermoeden dat het iets met database heeft die vol zit, krijg namelijk de volgende meldingen:

   Database growing large: 8988 MB, consider data cleanup (if not already enabled)   (now)
   Too many unprocessed readings: 109975   (2 days, 13 hours ago)
   Day statistics are lagging behind   (3 days, 16 hours ago)

Iemand tips hoe ik dit zou kunnen troubleshooten/oplossen?

### DSMR-reader version

DSMR-reader v4.19

### DSMR-reader platform

Native (e.g. manual installation)

### Debug info dump

```shell
DSMR-READER
    App / Python / Database                                                      v4.19 / v3.8.10 / postgresql
    BE sleep / DL sleep / Retention / Override                                     1.0s / 2.0s / 672h / False
    Latest telegram version read / Parser settings                                                 "50" / "4"

DATA
    Telegrams total (est.)                                                                            1288472
    Consumption records electricity / gas (est.)                                                65464 / 42559

UNRESOLVED ISSUES
    Database growing large: 8988 MB, consider data cleanup (if not already enabled)                                     now
    Too many unprocessed readings: 101727                                                 2 days, 8 hours ago
    Day statistics are lagging behind                                                    3 days, 16 hours ago

POSTGRESQL SIZE OF LARGEST TABLES (> 500 MB)
    public.dsmr_datalogger_dsmrreading                                                                8628 MB
```

**Comment by dennissiemensma** [comment, marked as answer]
Bedankt voor je melding, ik denk dat je als eerste dit eenmalig kunt proberen: https://github.com/dsmrreader/dsmr-reader/issues/1516#issuecomment-974709380

Het zou sowieso de meldingen over de databasegrootte moeten oplossing.
